src/utils/MastodonUtils.C

```diff
@@ -1,5 +1,12 @@
 // MASTODON includes
 #include "MastodonUtils.h"
+#include "MooseUtils.h"
+#include "math.h"
+#include <numeric>
+#include <algorithm>
+#include <boost/math/distributions.hpp>
+// #include </usr/local/boost_1_64_0/boost/math/distributions/lognormal.hpp>
+// #include </usr/local/boost_1_64_0/boost/math/distributions/fwd.hpp>
```

Comment:
If libmesh is not compiled with boost libraries, would only the probability tests fail or Mastodon would not successfully compile because the header file is not present? I am just wondering whether I (and others who have installed Mastodon) need to update libmesh once this PR is merged. If running the libmesh script is mandatory for Mastodon to work, then probably its better to send out an email in the mastodon-users group once the PR gets merged.

Reply:
@sveerara good point. I made a change and included 'BoostDistributions.h' from MOOSE in the header, instead of directly including a Boost header. This file creates dummy distributions if Boost is not included in libmesh. MASTODON will now compile without Boost, but some unit tests will fail. In the subsequent PRs that I am going to submit, some PRA-related tests will fail. 
@aeslaughter and I talked about this a while ago and we decided to rely on Boost for statistical distributions and the corresponding functions (cdf, pdf, etc.). I think we will have to rely on it more in the PRA and optimization project I will be working on this year.

Reply:
Thanks for checking the boost header issue. The one you and @aeslaughter came up with looks good.